components/card/test/card-footer-link.visual-diff.html

```diff
@@ -14,30 +17,47 @@
 			margin: 0;
 		}
 		.visual-diff {
+			padding-bottom: 3rem;
 			width: 300px;
 		}
 	</style>
 </head>
 <body class="d2l-typography">
 
 	<div class="visual-diff">
-		<d2l-card-footer-link id="no-secondary" icon="tier1:gear" text="Settings" href="https://www.d2l.com"></d2l-card-footer-link>
+		<d2l-card-footer-link id="no-secondary" icon="tier1:gear" text="Settings" href="https://www.d2l.com">
+			<d2l-tooltip slot="tooltip" for="no-secondary">No new settings</d2l-tooltip>
+		</d2l-card-footer-link>
 	</div>
 
 	<div class="visual-diff">
-		<d2l-card-footer-link id="secondary-notification" icon="tier1:gear" text="Settings" href="https://www.d2l.com" secondary-text="99+" secondary-text-type="notification"></d2l-card-footer-link>
+		<d2l-card-footer-link id="secondary-notification" icon="tier1:gear" text="Settings" href="https://www.d2l.com" secondary-count="100" secondary-count-type="notification">
+			<d2l-tooltip slot="tooltip" for="secondary-notification">100 new settings</d2l-tooltip>
+		</d2l-card-footer-link>
 	</div>
 
 	<div class="visual-diff">
-		<d2l-card-footer-link id="secondary-count" icon="tier1:gear" text="Settings" href="https://www.d2l.com" secondary-text="99+" secondary-text-type="count"></d2l-card-footer-link>
+		<d2l-card-footer-link id="secondary-count" icon="tier1:gear" tooltip-text="text here" text="Settings" href="https://www.d2l.com" secondary-count="100" secondary-count-max-digits="2" secondary-count-type="count">
+			<d2l-tooltip slot="tooltip" for="secondary-count">100 settings applied</d2l-tooltip>
+		</d2l-card-footer-link>
 	</div>
 
 	<div class="visual-diff">
-		<d2l-card-footer-link id="secondary-notification-rtl" icon="tier1:gear" text="Settings" href="https://www.d2l.com" secondary-text="99+" secondary-text-type="notification" dir="rtl"></d2l-card-footer-link>
```

Comment:
I've removed the rtl tests in card-footer-link and added some in the count-badge directly. Unfortunately, visual diff tests don't play nicely with nested rtl elements.